R/tools.R

```diff
@@ -26,28 +26,49 @@ lint_dir <- function(path) {
   lints
 }
 
+lint_path <- function(path) {
+  if (fs::is_dir(path)) {
+    lint_dir(path)
+  } else {
```

Comment:
Perhaps move the `else` branch to a new `lint_dir` function? I think having the very similar `lint_file` and `lint_dir` next to each other will make the code even more readable :slightly_smiling_face: